koku/koku/middleware.py

```diff
@@ -98,12 +103,17 @@ def __init__(self, dikt):
 
 class KokuTenantMiddleware(BaseTenantMiddleware):
     """A subclass of the Django-tenant-schemas tenant middleware.
-
     Determines which schema to use based on the customer's schema
     found from the user tied to a request.
     """
 
+    def __init__(self, get_response=None):
+        """ Create the tenant cache"""
+        super().__init__(get_response)
+        self.tenant_cache = TTLCache(maxsize=MAX_CACHE_SIZE, ttl=TIME_TO_CACHE)
```

Comment:
I don't know how cachetools works, but, is this okay as an instance variable? My gut tells me it should be a class attribute that all instances of the middleware share.

Reply:
I think that either would work based on my understanding. The tenant and customer cache are not being created with every request as that would defeat the purpose of the cache. Technically there would be more tenant and customer cache than the one user class though. I can change these to be class attribute so there is only one instance of each if that is preferred!